src/helpers/Templates.js

```diff
@@ -159,6 +159,164 @@ class Templates {
   </div>
     `;
   }
+
+  /**
+   *
+   *
```

Comment:
Good job @kayroy247. In the JSDOC comments for these methods, kindly include what each of these static methods does?

Reply:
Thanks for the feedback I will make changes now.